fr-French/strings.xml

```diff
@@ -1,301 +1,301 @@
-    <string name="activated">activated</string>
-    <string name="deactivate">deactivate</string>
-    <string name="deactivated">deactivated</string>
-    <string name="cancel">Cancel</string>
-    <string name="close">Close</string>
-    <string name="confirm">Confirm</string>
-    <string name="copy">Copy</string>
-    <string name="copied">Copied</string>
-    <string name="create">Create</string>
-    <string name="created">Created</string>
-    <string name="delete">Delete</string>
-    <string name="done">Done</string>
-    <string name="modify">Modify</string>
-    <string name="retry">Retry</string>
-    <string name="save">Save</string>
+    <string name="activate">activer</string>
+    <string name="activated">activé</string>
+    <string name="deactivate">désactiver</string>
+    <string name="deactivated">désactivé</string>
+    <string name="cancel">Annuler</string>
+    <string name="close">Fermer</string>
+    <string name="confirm">Confirmer</string>
+    <string name="copy">Copier</string>
+    <string name="copied">Copié</string>
+    <string name="create">Créer</string>
+    <string name="created">Créé</string>
+    <string name="delete">Supprimer</string>
+    <string name="done">Terminé</string>
+    <string name="modify">Modifier</string>
+    <string name="retry">Réessayer</string>
+    <string name="save">Sauvegarder</string>
     <!-- Left menu -->
     <string name="alias">alias</string>
-    <string name="aliases">aliases</string>
-    <string name="mailbox">mailbox</string>
-    <string name="mailboxes">mailboxes</string>
-    <string name="settings">Settings</string>
-    <string name="about">About</string>
-    <string name="rate_us">Rate Us</string>
-    <string name="sign_out">Sign Out</string>
-    <string name="sign_out_confirmation">You will be signed out</string>
-    <string name="yes_sign_me_out">Yes, sign me out</string>
+    <string name="aliases">alias</string>
+    <string name="mailbox">boîte mails</string>
+    <string name="mailboxes">boîtes mails</string>
+    <string name="settings">Réglages</string>
+    <string name="about">À propos</string>
+    <string name="rate_us">Notez l\'application</string>
+    <string name="sign_out">Se déconnecter</string>
+    <string name="sign_out_confirmation">Vous allez vous déconnecter</string>
+    <string name="yes_sign_me_out">Oui, déconnectez-moi</string>
     <!-- Alias -->
-    <string name="forward">forward</string>
-    <string name="forwards">forwards</string>
-    <string name="block">block</string>
-    <string name="blocks">blocks</string>
-    <string name="reply">reply</string>
-    <string name="replies">replies</string>
-    <string name="send_email">Send email</string>
-    <string name="deleted_alias">Deleted alias "%1$s"</string>
+    <string name="forward">transfert</string>
+    <string name="forwards">transferts</string>
+    <string name="block">bloqué</string>
+    <string name="blocks">bloqués</string>
```

Comment:
Singular for "Forward" and "Block" is never used, even if the counter is set to 0 or 1 (it's ok for reply/replies)